src/playground/with-layout/select/select-showcase.component.html

```diff
@@ -1,7 +1,8 @@
 <nb-select placeholder="Select Showcase" [(selected)]="selectedItem">
+  <nb-option value="">Option empty</nb-option>
+  <nb-option value="0">Option 0</nb-option>
   <nb-option value="1">Option 1</nb-option>
   <nb-option value="2">Option 2</nb-option>
   <nb-option value="3">Option 3</nb-option>
   <nb-option value="4">Option 4</nb-option>
 </nb-select>
-
```

Comment:
Please revert new line change.
By the way, to maintain consistent code style and prevent such changes we have `.editorconfig` file in our repo. If you'd like, you could install [editorconfig plugin](https://editorconfig.org/#download) for your editor or IDE, so it will automatically follow rules defined in project config, such as new line at the end of file.